src/doc/usage.md

```diff
@@ -27,7 +27,7 @@ HTML5 Boilerplate is a starting point, not a destination.
 
 A basic HTML5 Boilerplate site initially looks something like this:
 
-```
+``` bash
```

Comment:
is there any advantage/benefit to adding bash here? Does it change the color formatting in some way? If not, I think we could leave it out...

Reply:
👍